app/js/instances/containerController.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * instances/containerController.js
+ * Manages interactions for a single Docker container
+ */
+
+function containerController($scope, $routeParams, dockerService, instanceModel) {
+    'use strict';
+
+    $scope.host = $routeParams.host;
+    $scope.id = $routeParams.id;
+
+    $scope.$listenTo(instanceModel, function () {
+        $scope.info = instanceModel.getContainer($scope.id);
+
+        if ($scope.info) {
+            var dState = $scope.info.State;
+            $scope.state = {
+                running: dState.Running && !dState.Paused,
```

Comment:
This is a slight workaround. Docker considers a paused container be both paused *and* running within its state model. It makes sense, but doesn't bode well with the needs of the UI.